Tasks/MavenV3/maventask.ts

```diff
@@ -565,4 +568,20 @@ function execBuildWithRestore() {
         execBuild();
     }
 }
+
+function replaceGifToBase64() {
+    try {
+        const htmlString = fs.readFileSync(path.join(reportDirectory, 'index.html'), 'utf-8');
+        const dom = new JSDOM(htmlString);
+        const images = [...dom.window.document.getElementsByTagName('img')];
+        images.forEach(element => {
+            const file = fs.readFileSync(path.join(reportDirectory, element.src), 'base64')
+            element.src = 'data:image/gif;base64,' + file;
```

Comment:
Why it is always gif here?

Reply:
It's not always gif, that's why I added type into source of image tag.